ibm/service/vpc/resource_ibm_is_instance_template.go

```diff
@@ -75,6 +77,14 @@ func ResourceIBMISInstanceTemplate() *schema.Resource {
 		),
 
 		Schema: map[string]*schema.Schema{
+			isInstanceTemplateAvailablePolicyHostFailure: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
```

Comment:
Can you run one scenario create a template using old binary and then use this new binary and run Terraform plan and check if plan is showing any diff (both for instance, template you do this testing)

Reply:
TF plan says No changes. Infrastructure is up-to-date.